lib/vagrant-service-manager/command.rb

```diff
@@ -86,8 +87,20 @@ def check_if_a_service_is_running?(service)
 
       def print_all_provider_info
         @env.ui.info I18n.t('servicemanager.commands.env.nil')
-        self.execute_docker_info
-        self.execute_openshift_info
+
+        running_services = []
+        SUPPORTED_SERVICES.each do |service|
```

Comment:
> Capitalization of service name

`openshift` will be capitalized as `Openshift`, which ideally should be `OpenShift`. Instead of that here the service name is matching with the systemd name of the service.

> One time usage of @env.ui.info

Although `@env.ui.info` is written twice, but executed once per service in loop iteration and we are adding one more variable `status` in your suggestion! :-)